retriever/lib/datasets.py

```diff
@@ -15,6 +18,9 @@ def datasets(arg_keyword=None):
                     keywords = keywords + ',' + ','.join(script.keywords)
                 if keywords.lower().find(arg_keyword.lower()) != -1:
                     all_scripts.append(script)
+            elif licenses:
+                if str(script.licenses[0]['name']).lower() in licenses:
+                    all_scripts.append(script)
```

Comment:
when it comes to the interface, we can be more explicit and give options `datasets(arg_keyword=None, licenses=None):`

In this case let us not consider one option. 
since we have two possible options, we can just add the list instead of the mutual design (if else).

The overall interface design would be able to get me a list of either keywords or licenses or both.

Reply:
according to #969 `datasets` accepts only one keyword. I think I should change it to accept list of keywords. right ?

Reply:
If we want to accept many keywords then I guess we can change both the CLI and interface to consider that